bgc_data/combined_bgc_data.sql

```diff
@@ -54,7 +56,16 @@ UNION
    FROM bgc_chemistry_data che
 )
    SELECT
-      bm.*,
+      bm."Project",
+      bm."StationName",
+      bm."TripCode",
+      CASE WHEN che."SampleTime_Local" IS NOT NULL THEN che."SampleTime_Local"
+           WHEN (che."SampleTime_Local" IS NULL) AND (pig."SampleTime_Local" IS NULL) AND (tss."SampleTime_Local" IS NULL) THEN ppl."SampleTime_Local"
+           WHEN (che."SampleTime_Local" IS NULL AND pig."SampleTime_Local" IS NULL) THEN tss."SampleTime_Local"
+           WHEN che."SampleTime_Local" IS NULL THEN pig."SampleTime_Local" END AS "SampleTime_Local",
```

Comment:
Remember the `COALESCE` function!
```suggestion
    COALESCE(che."SampleTime_Local", pig."SampleTime_Local", tss."SampleTime_Local", ppl."SampleTime_Local") AS "SampleTime_Local",
```